### ISSUE TITLE
Remove the words "Command is" from the list of keyboard keys

### ISSUE THREAD

**Issue by jhomme** [microsoft/vscode#95993]
To make it easier to quickly scroll through the keyboard shortcuts list, if the words "Command is" were removed, the list entries  would start with many letters other than "C." This would make it so that a keyboard user could type various letters and instantly scroll to commands that start with that letter, making looking through this list much quicker than it is now. Since every line starts with "C," the keyboard user must use arrows, PageUp, PageDown, Home, and End, which slows productivity.

**Comment by isidorn** [comment]
@jhomme thanks for filling this issue. I have pushed a commit to make the aria labels in the keybindings editor more concise. Try it out with vscode insiders tomorrow and let me know how it behaves for you.

fyi @sandy081

**Comment by akash07k** [comment]
@isidorn 
Thanks for fixing this.
I got an insider update sometime back but it didn't fixed the issue. is the update with this fix not released as of now?

**Comment by isidorn** [comment]
Correct. There is still no release with this fix, it should be out soon
